pkg/project/util/util_test.go

```diff
@@ -0,0 +1,43 @@
+package util_test
```

Comment:
keep in the same `util` package... we don't typically set up special test packages unless we have to in order to break a cyclic dependency